src/braket/aws/aws_session.py

```diff
@@ -138,6 +138,19 @@ def _notebook_instance_version():
             f"{self.braket_client._client_config.user_agent} {additional_user_agent_fields}"
         )
 
+    def add_braket_user_agent(self, user_agent: str) -> None:
+        """
+        Appends the `user-agent` value to the User-Agent header, if it does not yet exist in the
+        header. This method is typically only relevant for libraries integrating with the
+        Amazon Braket SDK.
+
+        Args:
+            user_agent (str): The user_agent value to append to the header.
+        """
+        existing_user_agent = self.braket_client._client_config.user_agent
+        if user_agent not in existing_user_agent:
+            self.braket_client._client_config.user_agent = f"{existing_user_agent} {user_agent}"
```

Comment:
For my knowledge, does the order here matter? Is there a reason to append vs prepend?

Reply:
No, the order doesn't matter.